### PR TITLE
Charles android navbar

### PR DESCRIPTION
**Pull request by charlesx2013**

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: cd watershed-rails; rails s
```
Comment (nishdesai) [inline Procfile]: We don't need this file anymore

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Views/CircleImageView.java [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+     * setImageBitmap(android.graphics.Bitmap) and BitmapFactory instead."
+     * @param resId
+     */
+    @Override
+    public void setImageResource(int resId) {
+        super.setImageResource(resId);
+        isStateChanged = true;
+    }
+
+}
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Views/CircleImageView.java]: hehe reduce, reuse recycle. Can you go through this and clean up the commented code?

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/BaseRequest.java [hunk 3]
```diff
@@ -30,8 +30,8 @@
     private Response.Listener listener;
     private Response.Listener errorListener;
 
-//    private static final String baseURL = "http://192.168.0.111:3000/api/v1/";
-    private static final String baseURL = "https://intense-reaches-1457.herokuapp.com/api/v1/";
+    private static final String baseURL = "http://192.168.0.111:3000/api/v1/";
+//    private static final String baseURL = "https://intense-reaches-1457.herokuapp.com/api/v1/";
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Networking/BaseRequest.java]: also this...

File: watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Activities/MainActivity.java [hunk 4]
```diff
@@ -118,14 +132,37 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        authToken = prefs.getString("auth_token", "none");
-        authEmail = prefs.getString("auth_email", "none");
         mTitle = "Tasks";
 
     }
 
+    private void setUserObject() {
+        String userObject = mPreferences.getString("user", "none");
+        if (!userObject.equals("none")) getUserFromPreferences(userObject);
+        else makeHomeRequest();
```
Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Activities/MainActivity.java]: pls use brackets

Comment (charlesx2013) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Activities/MainActivity.java]: y

Comment (nishdesai) [inline watershed-android/watershed/app/src/main/java/com/blueprint/watershed/Activities/MainActivity.java]: readability

**Pull request merged**